docs/style-guide/lists/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Style Guide Documentation — Lists
+lang: en-US
+sampleListOne:
+  - title: Getting up to speed on Ethereum
+    subtitle: Matt Condon
+    meta: Aug 7, 2017
+    link: https://medium.com/@mattcondon/getting-up-to-speed-on-ethereum-63ed28821bbe
+  - title: Ethereum In Depth, Part 1
+    subtitle: Facu Spagnuolo
+    meta: May 11, 2018
+    link: https://blog.openzeppelin.com/ethereum-in-depth-part-1-968981e6f833/
+  - title: Ethereum In Depth, Part 2
+    subtitle: Facu Spagnuolo
+    meta: July 24, 2018
+    link: https://blog.openzeppelin.com/ethereum-in-depth-part-2-6339cf6bddb9/
+sampleListTwo:
+  - title: EEA
+    subtitle: The Enterprise Ethereum Alliance is a member-driven standards organization whose charter is to develop open, blockchain specifications that drive harmonization and interoperability for businesses and consumers worldwide. Our global community of members is made up of leaders, adopters, innovators, developers, and businesses who collaborate to create an open, decentralized web for the benefit of everyone.
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    subtitle: Hyperledger is an open source collaborative effort created to advance cross-industry blockchain technologies. It is a global collaboration, hosted by The Linux Foundation, including leaders in finance, banking, Internet of Things, supply chains, manufacturing and Technology. The foundation has some projects in it that work with the Ethereum stack.
+    link: https://hyperledger.org/
+sampleListThree:
+  - title: EEA
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    link: https://hyperledger.org/
+sampleListFour:
+  - title: EEA
+    meta: John Owens
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    meta: John Owens
+    link: https://hyperledger.org/
+---
+
+# Component Usage
+
+## Standard Lists
+
+Standard lists can be called anywhere in markdown by using
+
+```
+- Lorem ipsum
+- Si dolor et amet
+- nunc habitasse portitor demi
+```
+
+Ensure that the list is always preceeded with one of the following, taking care to ensure that your markup is semantic:
```

Comment:
I don't understand - why must every list be preceded by a list title?

Reply:
It's in the Figma Style Guide. I've not included it in the component itself as I imagine we might need to drop it when updating content.